Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/VesuvioDiffractionReduction.py

```diff
@@ -25,7 +25,7 @@ def category(self):
         return 'Diffraction\\Reduction'
 
     def summary(self):
-        return 'Performs diffraction reduction for VESUVIO'
+        return ('Performs diffraction reduction for VESUVIO. This algorithm is deprecated (April-2017).')
```

Comment:
This is good but I think we should also issue a Python deprecation warning in `PyExec`:

```python
import warnings
warnings.warn("A message", DeprecationWarning)
```